src/engraving/types/typesconv.cpp

```diff
@@ -1607,8 +1607,8 @@ static const std::vector<Item<BeamMode> > BEAMMODE_TYPES = {
     { BeamMode::MID, "mid" },
     { BeamMode::END, "end" },
     { BeamMode::NONE, "no" },
-    { BeamMode::BEGIN32, "begin32" },
-    { BeamMode::BEGIN64, "begin64" },
+    { BeamMode::BEGIN16, "begin32" },
+    { BeamMode::BEGIN32, "begin64" },
```

Comment:
This conversion looks strange. Can we keep it only for the old file readers and use the new name for the 410 reader and writer?

Reply:
Well, the 410 files still use `begin32` and `begin64`-- I didn't touch the file format whatsoever, so I'm not sure what you mean by this

Reply:
Roman means that you _should_ touch the file format, so that 410 files will not use these janky names anymore, but old files (that do still use them) keep working

Reply:
Oh, well, this one seems to be rather important for 4.1, do we really want to introduce file format changes in this one?

Reply:
I'll look into it!

Reply:
Looks better! We just need to make the same change for other readers (at least for read206)